### PR TITLE
Single place of input modifications

### PR DESCRIPTION
**Pull request by danielpeintner**
fixes https://github.com/eclipse/thingweb.node-wot/issues/427
~~fixes https://github.com/eclipse/thingweb.node-wot/issues/426 for v.08.x~~

### CODE REVIEW

File: packages/core/src/exposed-thing.ts [hunk 1]
```diff
@@ -67,9 +67,48 @@ export default class ExposedThing extends TD.Thing implements WoT.ExposedThing {
...
+                    if (arr[i] === "tm:ThingModel") {
+                        arr.splice(i, 1);
+                        i--;
+                    }
+                }
+            }
+        }
+        // set default language
+        this.addDefaultLanguage(this);
+        // extend interactions
```
Comment (relu91) [inline packages/core/src/exposed-thing.ts:87]: I would approve the change but see my comment: https://github.com/eclipse/thingweb.node-wot/issues/426#issuecomment-859660742

**Review by relu91 — COMMENTED** [review]
PR looks good but sadly we have to wait until the discussion on https://github.com/w3c/wot-scripting-api/issues/304 is settled

**Comment by danielpeintner** [comment]
I think this PR should resolve #427.

Having said that, we might need to continue discussing #426.
I kept the _current_ code for #426 in, by commenting it out. Not sure whether we should totally remove it as part of this PR. Any opinion?

**Review by relu91 — APPROVED** [review]
Yeah, I agree, I would even uncomment the tests cause they provide useful information anyway. But it is fine, we will fix it when #426 is settled

**Comment by danielpeintner** [comment]
@relu91 thanks for the feedback. Let's merge this PR and resolve #426 related issues in a later PR (the *initial* code and the commented tests are there...)

**Pull request merged**